account_banking_pain_base/models/account_payment_order.py

```diff
@@ -100,15 +100,18 @@ def _prepare_field(self, field_name, field_value, eval_ctx,
                 for unallowed_ascii_char in unallowed_ascii_chars:
                     value = value.replace(unallowed_ascii_char, '-')
         except:
-            line = eval_ctx.get('line')
-            if line:
-                raise UserError(
-                    _("Cannot compute the '%s' of the Payment Line with "
-                        "reference '%s'.")
-                    % (field_name, line.name))
-            else:
-                raise UserError(
-                    _("Cannot compute the '%s'.") % field_name)
+            error_msg_list = self.except_messages_prepare_field(
+                eval_ctx, field_name)
+            if error_msg_list:
+                raise UserError('\n'.join(error_msg_list))
```

Comment:
I'm thinking that this is not needed. Only append this to the general one, removing redundant part.